### ISSUE TITLE
Links on the "Nodes" table on "Overview" don't go to the correct place

### ISSUE THREAD

**Issue by ziggythehamster** [scylladb/scylla-monitoring#1130]
In Scylla Monitoring 3.5's Overview dashboard, the Nodes table has handy links. Unfortunately, they don't work, at least with 2020.1. The patch from #1124 has no effect on either the alerts or the nodes list. This is a reopening of #1122.

It is also worth mentioning that the Grafana log (`docker logs agraf`) has this repeated every 10 seconds (and nobody is saving any dashboards, so I'm not sure why it cares.. maybe Grafana always tries to snapshot state or something):

```
t=2020-11-10T22:47:54+0000 lvl=eror msg="failed to save dashboard" logger=provisioning.dashboard type=file name=2020.1 error="uid contains illegal characters"
```

Going back to the nodes list, the CQL link is:

```
http://foo:3000/d/cql-2020.1/scylla-cql?refresh=30s&orgId=1&var-by=instance&var-node=10.20.7.16
```

This should be:

```
http://foo:3000/d/cql-2020-1/scylla-cql?refresh=30s&orgId=1&var-by=instance&var-node=10.20.7.16
```

The node detail link is:

```
http://foo:3000/d/detailed-2020.1/Detailed?refresh=30s&orgId=1&var-by=instance&var-node=10.20.7.16
```

It should be:

```
http://foo:3000/d/detailed-2020-1/detailed?refresh=30s&orgId=1&var-by=instance&var-node=10.20.7.16
```

And finally, the version link is:

```
http://foo:3000/d/OS-2020.1/OS-metrics?refresh=30s&orgId=1&var-by=instance&var-node=10.20.7.16
```

It should be:

```
http://foo:3000/d/OS-2020-1/os-metrics?refresh=30s&orgId=1&var-by=instance&var-node=10.20.7.16
```

I didn't record what the alerts links did before they disappeared, but they didn't work with probably the same issue.

**Comment by amnonh** [comment]
Can you make sure it is still not working with 3.5.1?

**Comment by ziggythehamster** [comment]
Didn't see that it was released - 3.5.1 does work, and also doesn't have the weird logging issue. Not sure why the #1124 patch didn't work.
